### ISSUE TITLE
FTX - orderbook no trades 

### ISSUE THREAD

**Issue by Dante2333** [freqtrade/freqtrade#3005]
It seems dogehedge pair does not have any orderbook trades in the last 24 hrs, ccxt queries and gets a NoneType which then yields an error. 

Using market order, latest dev

```
2020-03-01 01:17:28,780 - freqtrade.wallets - INFO - Wallets synced.
2020-03-01 01:17:28,782 - freqtrade.freqtradebot - INFO - Buy signal found: about create a new trade with stake_amount: 108 ...
2020-03-01 01:17:28,782 - freqtrade.freqtradebot - INFO - Executing Buy for DOGEHEDGE/USD
2020-03-01 01:17:28,782 - freqtrade.freqtradebot - INFO - Using Last Ask / Last Price
2020-03-01 01:17:28,859 - freqtrade.commands.trade_commands - INFO - worker found ... calling exit
2020-03-01 01:17:28,859 - freqtrade.rpc.rpc_manager - INFO - Sending rpc message: {'type': status, 'status': 'process died'}
2020-03-01 01:17:29,229 - freqtrade.freqtradebot - INFO - Cleaning up modules ...
2020-03-01 01:17:29,229 - freqtrade.rpc.rpc_manager - INFO - Cleaning up rpc modules ...
^[[C^[[C2020-03-01 01:17:52,307 - freqtrade - ERROR - Fatal exception!
Traceback (most recent call last):
  File "/home/dante/freqtrade/freqtrade/main.py", line 36, in main
    return_code = args['func'](args)
  File "/home/dante/freqtrade/freqtrade/commands/trade_commands.py", line 20, in start_trading
    worker.run()
  File "/home/dante/freqtrade/freqtrade/worker.py", line 66, in run
    state = self._worker(old_state=state)
  File "/home/dante/freqtrade/freqtrade/worker.py", line 104, in _worker
    self._throttle(func=self._process_running, throttle_secs=self._throttle_secs)
  File "/home/dante/freqtrade/freqtrade/worker.py", line 125, in _throttle
    result = func(*args, **kwargs)
  File "/home/dante/freqtrade/freqtrade/worker.py", line 139, in _process_running
    self.freqtrade.process()
  File "/home/dante/freqtrade/freqtrade/freqtradebot.py", line 151, in process
    self.enter_positions()
  File "/home/dante/freqtrade/freqtrade/freqtradebot.py", line 217, in enter_positions
    trades_created += self.create_trade(pair)
  File "/home/dante/freqtrade/freqtrade/freqtradebot.py", line 413, in create_trade    return self.execute_buy(pair, stake_amount)
  File "/home/dante/freqtrade/freqtrade/freqtradebot.py", line 453, in execute_buy
    buy_limit_requested = self.get_buy_rate(pair, True)
  File "/home/dante/freqtrade/freqtrade/freqtradebot.py", line 249, in get_buy_rate    if ticker['ask'] < ticker['last']:
TypeError: '<' not supported between instances of 'float' and 'NoneType'
```

**Comment by xmatthias** [comment]
That's technically untrue.
last is None - yes ... 




```
{'symbol': 'DOGEHEDGE/USD',
 'timestamp': 1583174908416,
 'datetime': '2020-03-02T18:48:28.416Z',
 'high': None,
 'low': None,
 'bid': 6263.5,
 'bidVolume': None,
 'ask': 6299.5,
 'askVolume': None,
 'vwap': None,
 'open': None,
 'close': None,
 'last': None,
 'previousClose': None,
 'change': None,
 'percentage': -0.06695384158342307,
 'average': None,
 'baseVolume': None,
 'quoteVolume': 0.0,
 'info': {'ask': 6299.5,
  'baseCurrency': 'DOGEHEDGE',
  'bid': 6263.5,
  'change1h': 0.005442176870748299,
  'change24h': -0.06695384158342307,
  'changeBod': -0.04554605887939221,
  'enabled': True,
  'last': None,
  'name': 'DOGEHEDGE/USD',
  'price': 6281.5,
  'priceIncrement': 0.5,
  'quoteCurrency': 'USD',
  'quoteVolume24h': 0.0,
  'sizeIncrement': 1e-05,
  'type': 'spot',
  'underlying': None,
  'volumeUsd24h': 0.0}}
```

but i'm not sure if this isn't just one of the "oddities" of FTX - which also doesn't show volume for leveraged pairs even though there were trades ...

In my understanding of candles - price moves only if a trade has been filled. For this pair, candles move but no trades seem to be filled ... which (as i think i've said before) - smells like market manipulation to me - but maybe it's just the way the leveraged tokens work (?) 

either way - i suspect that's not a trade you'd want to take anyway - spread is currently >40 USD ... so spread alone will eat you alive ... 

Honestly, if you're keen on trading such pairs - please enable orderbook for both sides of the orderbook (bid_strategy / ask_strategy).
You can take a look at how buy side pricing works [here](https://www.freqtrade.io/en/latest/configuration/#buy-price-without-orderbook-enabled) ... it takes a position "somewhere" between ask and last ... so for an asset without Last - that would not work.


So the solution to this is to not raise an error (i assume the above crashed the bot) - but to raise a controlled exception to prevent the buy, since incomplete data in this case suggests problems either with the exchange, or with the underlying asset.

**Comment by Dante2333** [comment]
its not urget by any means. This pair did not have any trades in the last 24hrs, could be that it did not have any trades in the last 3 days? who knows, would have to verify... Do you want me to test using bidstrategy? remember im using market orders.

**Comment by xmatthias** [comment]
the order type is not relevant ... this is about generating the price - which is still necessary also for market orders (to get the amount to buy in stake-currency).

as said, i can see the error - and it'll be fixed once #2982 is merged...
